src/lib/chains.ts

```diff
@@ -21,6 +21,9 @@ export const chainsNames = [
   'moonbeam',
   'optimism',
   'polygon',
+  'arbitrum-nova',
```

Comment:
can you remove `arbitrum-nova` and `bittorrent` please ? we don't really support them atm

Reply:
Sure, on my way